Makefile

```diff
@@ -85,6 +85,10 @@ test_c: export DD_TRACE_CLI_ENABLED=1
 test_c: $(SO_FILE) $(TEST_FILES) $(TEST_STUB_FILES)
 	$(RUN_TESTS_CMD) -d extension=$(SO_FILE) $(BUILD_DIR)/$(TESTS)
 
+test_c_coverage: export DD_TRACE_CLI_ENABLED=1
```

Comment:
I think we do not need this line, since we are calling `make test_c` internally in any case and the sub-call to make will set it at the time tests are run. Unless I am making some wrong assumption

Reply:
Good catch